.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,14 +28,21 @@ For refactoring and code cleanup changes, exercise the code before and after the
 
 ### Proposed changelog entries
 
-- Entry 1: Issue, human-readable text
-- […]
-
-<!-- Comment:
+<!--
+Please propose a changelog entry (or entries) for this pull request below between the lines with three backticks.
+Add more than one entry if necessary, but be aware that this is usually an indicator that a PR should be split into multiple PRs.
 The changelog entry should be in the imperative mood; e.g., write "do this"/"return that" rather than "does this"/"returns that".
+Use basic HTML like '<strong>', '<a href="…">', '<code>' to format changelog entries. Please do not use Markdown or Asciidoc.
+
 For examples, see: https://www.jenkins.io/changelog/
+
+Full changelog entry guideline: https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/_STYLEGUIDE.adoc#labels
 -->
 
+```
+- Improve spline reticulation.
+```
```

Comment:
The changelog generator works better when list items begin with `*` rather than `-`. List items that are started with `*` have worked the best in what I have seen.  When list items begin with `-`, the `-` is often included in the generated changelog message and needs to be removed by the docs SIG when reviewing and finalizing the changelog.

I'm not sure why we would put the proposed changelog entry in backticks.  Can you explain further why the backticks were added?  Is it intended that the pull request submitter will remove the backticks while creating the pull request?
```suggestion
* Improve spline reticulation.
```

Reply:
> Can you explain further why the backticks were added? Is it intended that the pull request submitter will remove the backticks while creating the pull request?

This prevents Markdown from rendering on the PR page, so it doesn't automatically look "good" in the PR (while being wrong for what we need).

Reply:
I hadn't thought of that.  That's a great technique!  If someone mistakenly uses markdown or asciidoc, the backticks will increase the chance they will realize that they made a mistake.

Reply:
any reason why you used 'Improve spline reticulation'?
```suggestion
```
- Replace this text with a proposed changelog entry.
```
```